e2e/cypress/integration/api/iam/01_projects_api.spec.ts

```diff
@@ -347,3 +345,21 @@ function cleanupTestProjects(id_token: string): void {
     }
   });
 }
+
+function waitUntilApplyRulesNotRunning(attempts: number): void {
```

Comment:
This should count _down_ I think. We'd pass `100`, and it would stop when that is reached. `waitUntilApplyRulesNotRunning(0);` looks quite counter-intuitive 😉

Reply:
That let's them pass in how many attempts they want as well. I like it.